### PR TITLE
[Clojure] Add missing files for basics exercise

### PR DESCRIPTION
**Pull request by porkostomus**

### CODE REVIEW

File: languages/clojure/exercises/concept/basics/.meta/design.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- Higher-order functions.
+- Anonymous functions.
+- Multi-arity and variadic functions.
+- Using functions from other namespaces
+- Partial application.
+- Organizing functions in modules and namespaces.
+
+## Concepts
+
+- `basics`: know what a value is; know how to define a binding; know that bindings are immutable; know how to define scope using significant whitespace; know that bindings require dependency order; know how to define a function; know how to return a value from a function; know how to call a function; know what type inference is; know how type inference works for bindings; know how to define an integer; know how to use mathematical operators on integers; Know how to define single- and multiline comments. - know what a value is; know how to define a var; know how to define a function; know how to return a value from a function; know how to call a function; know how to define an integer; know how to use mathematical operators on integers; know how to use single-line comments and comment forms.
```
Comment (bemself) [inline languages/clojure/exercises/concept/basics/.meta/design.md:26]:
```suggestion
- `basics`:  know what a value is; know how to define a var; know how to define a function; know how to return a value from a function; know how to call a function; know how to define an integer; know how to use mathematical operators on integers; know how to use single-line comments and comment forms.
```

**Comment by ErikSchierboom** [comment]
@porkostomus Is there anything I can do to help with this PR?

**Comment by porkostomus** [comment]
@ErikSchierboom This PR (as well as a few others, linked below) is basically done and seems to have stalled out while waiting for review. The following files are (at least mostly) complete:

- config.json
- design.md
- example.clj
- after.md
- hints.md
- instructions.md
- introduction.md
- basics_test.clj
- basics.clj

My intention was that this could be merged as part of an effort to make smaller PRs that are easier to review. The same applies to the following PRs:

- #1801 
- #1807 
- #1809 
- #1813 
- #1835

**Comment by ErikSchierboom** [comment]
@porkostomus Right! I'll go through all of them.

**Review by ErikSchierboom — APPROVED** [review]
LGTM! Thanks!

**Comment by ErikSchierboom** [comment]
I'm reviewing all your open PR's! So sorry for them being left unreviewed for quite a while.

**Pull request merged**